### PR TITLE
Quickverse develop

### PR DESCRIPTION
**Pull request by mithiladongre**
Please review the code changes and approve

### CODE REVIEW

File: src/Components/Login/SignUp.tsx [hunk 1]
```diff
@@ -153,6 +167,24 @@ const SignupScreen: React.FC = () => {
           maxLength={4}
         />
       </View>
+      <View style={styles.inputContainer}>
+        <MaterialCommunityIcons
+          name="lock"
```
Comment (Abhilashghope) [inline src/Components/Login/SignUp.tsx:172]: change the icon to ""lock-check"

File: src/Components/Login/SignUp.tsx [hunk 2]
```diff
@@ -204,7 +236,7 @@ const styles = StyleSheet.create({
   inputContainer: {
     flexDirection: 'row',
     alignItems: 'center',
-    height: 50,
+    height: 40,
```
Comment (Abhilashghope) [inline src/Components/Login/SignUp.tsx:239]: keep the original style

File: src/Components/Login/SignUp.tsx [hunk 3]
```diff
@@ -223,8 +255,8 @@ const styles = StyleSheet.create({
     overflow: 'hidden',
   },
   logo: {
-    width: 150,
-    height: 160,
+    width: 145,
```
Comment (Abhilashghope) [inline src/Components/Login/SignUp.tsx]: keep original style

File: src/Components/Login/SignUp.tsx [hunk 4]
```diff
@@ -153,6 +167,24 @@ const SignupScreen: React.FC = () => {
...
+      <View style={styles.inputContainer}>
+        <MaterialCommunityIcons
+          name="lock"
+          size={24}
+          color={theme.colors.ternary}
+          style={styles.icon}
+        />
+        <TextInput
+          style={styles.input}
+          placeholder="confirmation Pin"
```
Comment (Abhilashghope) [inline src/Components/Login/SignUp.tsx]: change placeholder name to "Confirm Pin"

File: src/Components/Login/SignUp.tsx [hunk 5]
```diff
@@ -39,13 +40,25 @@ const SignupScreen: React.FC = () => {
       isLoadingCampuses(false);
     }
   };
+  const handlingconfirmPin = (value: string) => {
+    setconfirmPin(value); 
+  
+    if (pin !== value) {
+      setError('Confirmation pin does not match the original pin');
```
Comment (Abhilashghope) [inline src/Components/Login/SignUp.tsx:47]: Change the verbiage to "Pin does not match with the original pin"

File: src/Components/Login/SignUp.tsx (hunk 2 above)
Comment (mithiladongre) [reply src/Components/Login/SignUp.tsx:239]: I did change the styling of the logo also so should I keep it as it is or change as per the original one

**Review by Abhilashghope — APPROVED** [review]

**Review by Abhilashghope — APPROVED** [review]

**Pull request merged**